### DISCUSSION TITLE
How to get info for unregistered datasets

### DISCUSSION THREAD

**Discussion by rachelmadang** [uclahs-cds/pipeline-align-DNA#245] [Q&A]
After using convert-BAM2FASTQ to convert my samples I want to use align-DNA but I don't have all of the information needed to fill all fields of the input.csv. Is there anything I can do about that or a way to find out some of the information for an unregistered dataset?

**Comment by jarbet** [comment]
Hi @forbiddenpersimmon ,  what are the fields from [here](https://github.com/uclahs-cds/pipeline-align-DNA/blob/main/input/align-DNA.input.csv) that you are unsure about?

**Reply by rachelmadang** [reply]
I am missing the sequencing_center, library_identifier, and lane fields. I was able to get the other information from the original BAM file (if this is not okay let me know). From the linked read groups page it looks like I can get the lane information from the PU?

**Reply by jarbet** [reply]
> I am missing the sequencing_center, library_identifier, and lane fields. I was able to get the other information from the original BAM file (if this is not okay let me know). From the linked read groups page it looks like I can get the lane information from the PU?

From discussion with @yashpatel6 :

> ...if the information can't be found in the FASTQ/BAM header, the next recommendation is to check any publications tied to the dataset (a lot of times, the metadata is in the paper or supplemental tables) and then potentially reach out to the original source of the data for the information if necessary. Things like sequencing_center can also usually be set based on where the data was obtained. Library's a bit trickier if the source doesn't provide it in any way; I've just gone with using some sort of run ID to generate a library ID. Lane should almost always be extractable from a FASTQ/BAM header

**Comment by tnyamaguchi** [comment]
@forbiddenpersimmon I suggest that you check out [this pinned issue](https://github.com/uclahs-cds/pipeline-align-DNA/issues/143) if you haven't. For your GBM dataset, we don't have to worry too much about the `sequencing_center` info (`NA` is fine).
